sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/KotlinTreeVisitor.java

```diff
@@ -230,6 +243,35 @@ private Tree createParenthesizedExpression(TreeMetaData metaData, KtParenthesize
     Token rightParenthesis = toSlangToken(element.getLastChild());
     return new ParenthesizedExpressionTreeImpl(metaData, expression, leftParenthesis, rightParenthesis);
   }
+  private Tree createReturnTree(TreeMetaData metaData, KtReturnExpression element) {
+    Tree returnExpression = null;
+    if (element.getReturnedExpression() != null) {
+      returnExpression = createElement(element.getReturnedExpression());
+    }
+    return new ReturnTreeImpl(metaData, toSlangToken(element.getReturnKeyword()), returnExpression);
+  }
+
+  private Tree createBreakTree(TreeMetaData metaData, KtBreakExpression ktBreakExpression) {
+    IdentifierTree label = null;
+
+    KtSimpleNameExpression targetLabel = ktBreakExpression.getTargetLabel();
+    if (targetLabel != null && targetLabel.getIdentifier() != null) {
+      PsiElement identifier = targetLabel.getIdentifier();
+      label = createIdentifierTree(getTreeMetaData(identifier), identifier.getText());
```

Comment:
IMO we should create metadata from `KtSimpleNameExpression` (so that label tokens contain `@` token), but that worths a discussion.
 Which triggers idea that we are missing the test for tokens available for `break@foo`